src/raudio.c

```diff
@@ -1958,12 +1958,12 @@ static int SaveWAV(Wave wave, const char *fileName)
     format.bitsPerSample = wave.sampleSize;
 
     unsigned char *fileData = NULL;
-    unsigned int fileDataSize = 0;
+    size_t fileDataSize = 0;
```

Comment:
In raylib I'm trying to avoid the `size_t` type... Please, would you mind leaving this as `unsigned int`?

Reply:
Yes I saw that and minimized the use of size_t, in this case it is required because drwav_init_memory_write is written to take a size_t as a pointer. There would still need to be a cast to size_t in the call at the same scope as the variable.

Reply:
I did a quick test of this function on a 64 bit build, and it does indeed corrupts the stack if called with an unsigned int as the fileDataSize parameter. I would guess that this function is not called very often so the problem has gone undiscovered.

![image](https://user-images.githubusercontent.com/322174/100556409-22a95b80-3257-11eb-8b75-582d50678213.png)

It think this is the only place where size_t is required, not just for warnings but for correct operation. size_t is must already be included because drwav_init_memory_write uses it as an argument, so I don't think you are going to have any problem with compiler support. Any compiler that doesn't support it won't compile the rest of this code anyway.

Reply:
Hi! Thanks for the detailed report and the testing! I try to avoid using `size_t` but this case seems unavoidable.